packages/web-components/src/components/va-checkbox-group/va-checkbox-group.scss

```diff
@@ -24,29 +24,31 @@
 @import '../../mixins/hint-text.css';
 @import '../../mixins/headers.css';
 
-:host(:not([uswds])) #error-message {
-  font-size: 1.6rem;
-  line-height: 2.4rem;
-  font-weight: 700;
-  margin-bottom: 0;
-}
-
-:host(:not([uswds])) .required {
-  color: var(--color-secondary-dark);
-  margin: 0 0.35rem;
-}
-
-:host(:not([uswds])) fieldset {
-  border: none;
-  margin: 0;
-  padding: 0;
-}
-
-:host(:not([uswds])) legend {
-  display: block;
-  max-width: 46rem;
-  font-weight: inherit;
-  font-size: inherit;
-  line-height: inherit;
-  padding-inline: 0;
+:host(:not([uswds='true'], [uswds=''])) {
```

Comment:
I haven't worked too much with the `:not()` selector, so I was surprised to discover the selector list passed in isn't a "cover all of these scenarios" sort of thing. It only uses whichever selector is the most specific. 

This led me down a deeper rabbit hole specificity weight and `:is()`. 

Could this selector also be written as `:host(:not(:is([uswds='false']))`? 

I don't think that's any clearer necessarily, just curious if it's the same thing.